### PR TITLE
Fixed issue around toggling custom toolbar items.

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: src/app/public/modules/list-toolbar/list-toolbar.component.ts [hunk 1]
```diff
@@ -165,6 +165,8 @@ export class SkyListToolbarComponent implements OnInit, AfterContentInit, OnDest
   private hasSortSelectors: boolean = false;
   private ngUnsubscribe = new Subject();
 
+  private _customItemIds: string[] = [];
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/list-toolbar/list-toolbar.component.ts]: Shouldn't the underscore convention only be used if controlled by a setter?

File: src/app/public/modules/list-toolbar/list-toolbar.component.ts [hunk 2]
```diff
@@ -317,6 +319,34 @@ export class SkyListToolbarComponent implements OnInit, AfterContentInit, OnDest
         ],
         toolbarItem.index
       );
+
+      this._customItemIds.push(toolbarItem.id);
+    });
+
+    this.toolbarItems.changes.subscribe((newItems: QueryList<SkyListToolbarItemComponent>) => {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/list-toolbar/list-toolbar.component.ts]: Should we unsubscribe from this observable?

File: src/app/public/modules/list-toolbar/list-toolbar.component.ts (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/list-toolbar/list-toolbar.component.ts]: Doh. Yes. I had it with a setter at one point and that the wasn't needed and I didn't get it changed back. Good catch. Fixed.

File: src/app/public/modules/list-toolbar/list-toolbar.component.ts (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/list-toolbar/list-toolbar.component.ts]: Good catch. Fixed.

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-AlexKingman ready for you again.

**Pull request merged**